### PR TITLE
Implementing basic routing

### PR DESCRIPTION
**Pull request by Tottoroo-Okarin**

### CODE REVIEW

File: client-react/src/App.tsx [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+  })
+
   return (
-    <div className={styles['App']}>
+    <div className={appStyles}>
       <Layout>
-        <TodoListContainer containerOptions={{hasActions: true}}/>
+        <Switch>
+          <Route exact path="/" component={TodoListPage} />
+          <Route path="/details/:id" component={TodoDetailsPage} />
```
Comment (MursalovAltun) [inline client-react/src/App.tsx:25]: Path name is not explanatory. What's details is this? Todo details? Can we make it like todos/details/id?

Add a route /todos and reroute from route '/' to 'todos' automatically everytime

File: client-react/src/features/todos/components/TodoDetailsContainer/TodoDetailsContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            <TodoDetailsComments todo={todo}/>
+          </div>
+          <div className={metaContentClasses}>
+            <TodoDetailsMetaInfo todo={todo}/>
+          </div>
+        </div>
+      </Card>
+    )} else {
+      return(
+        <h3>RIP</h3>
```
Comment (MursalovAltun) [inline client-react/src/features/todos/components/TodoDetailsContainer/TodoDetailsContainer.tsx:110]: Need to create a common 404 component to display instead

**Review by MursalovAltun — COMMENTED** [review]
See comments above

File: client-react/src/App.tsx (hunk 1 above)
Comment (Tottoroo-Okarin) [reply client-react/src/App.tsx]: Fixed

File: client-react/src/features/todos/components/TodoDetailsContainer/TodoDetailsContainer.tsx (hunk 2 above)
Comment (Tottoroo-Okarin) [reply client-react/src/features/todos/components/TodoDetailsContainer/TodoDetailsContainer.tsx]: Fixed

File: client-react/src/features/todos/components/TodoModal/TodoModal.tsx [hunk 3]
```diff
@@ -44,8 +62,12 @@ export const TodoModal = ({isOpen}: TodoModalProps) => {
     dispatch(TodoModalsActions.closeTodoModal())
   } 
   
-  const addTodo = (title: string, description: string) => {
-    dispatch(TodosCollectionActions.add({ id: uuidv4(), title: title, description: description, completed: false }))
+  const addTodo = (title: string, description: string, estimate: number, assigneeId: string) => {
+    dispatch(TodosCollectionActions.add(
+      { id: uuidv4(), title: title, description: description,
```
[the remark this answers is not in the document]
Comment (Tottoroo-Okarin) [reply client-react/src/features/todos/components/TodoModal/TodoModal.tsx]: Fixed

**Review by MursalovAltun — APPROVED** [review]

**Pull request merged**